test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -52,6 +124,72 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	Describe("[rfe_id: 64006][Dynamic OVS Pinning]", Ordered, func() {
+		Context("[Performance Profile applied]", func() {
+			It("[test_id:64097] Activation file is created", func() {
+				cmd := []string{"ls", activation_file}
+				out, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+				Expect(err).ToNot(HaveOccurred(), "file %s doesn't exist ", activation_file)
+				Expect(out).To(Equal(activation_file))
+			})
+		})
+		Context("[Performance Profile Modified]", func() {
+			BeforeEach(func() {
+				initialProfile = profile.DeepCopy()
+			})
+			It("[test_id:64099] Activation file doesn't get deleted", func() {
+				performanceMCP, err := mcps.GetByProfile(profile)
+				Expect(err).ToNot(HaveOccurred())
+				policy := "best-effort"
+				// Need to make some changes to pp , causing system reboot
+				// and check if activation files is modified or deleted
+				profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+				Expect(err).ToNot(HaveOccurred(), "Unable to fetch latest performance profile")
+				currentPolicy := profile.Spec.NUMA.TopologyPolicy
+				if *currentPolicy == "best-effort" {
+					policy = "restricted"
+				}
+				profile.Spec.NUMA = &performancev2.NUMA{
+					TopologyPolicy: &policy,
+				}
+				By("Updating the performance profile")
+				profiles.UpdateWithRetry(profile)
```

Comment:
I see you revert the profile in aftereach, why not add a defer() here to revert it, it's only one test that modifies the profile. 
OTOH if you don't specifically need a PP update, then I believe system reboot on the node on which the pod is running can be more efficient here and cheaper.

Reply:
we can't run systemctl reboot from the pod, It doesn't allow 
For example:

```
[root@dell-r630-007 ~]# oc exec -it pods/machine-config-daemon-bvp6t -n openshift-machine-config-operator -- bash -c "systemctl reboot"
Defaulted container "machine-config-daemon" out of: machine-config-daemon, kube-rbac-proxy
Running in chroot, ignoring request: reboot
```
Also yes we are changing Profile to trigger reboot, but this is the most safest way to reboot as all the utility functions are well tested and used extensively.

Reply:
re: reboot, you can do jumping through some hoops: https://github.com/openshift/cluster-node-tuning-operator/blob/master/test/e2e/performanceprofile/functests/9_reboot/devices.go#L125

that said, can't tell if the reboot is the best approach here

Reply:
also executing commands through mcd pod is fine as long as you need some output, executing commands like "reboot", where you will instantly loose connection to the node , i find it not so appropriate. As i mentioned above, i can follow the same method if required.

Reply:
I see your point, but there is a big difference in execution time with the PP update vs node reboot. when PP is updated it will trigger reboot on all (usually not less than 2) nodes, and later on the revert will do the same, so the time is x4 than rebooting only the node on which the pod is running. I do not expect the pod to be terminated on node reboot, otherwise it would likely be a bug

Reply:
okay, so I see this involves all nodes. following this, correct me if mistaken,  I think it is a good idea to cover that also when verifying the existence of the file not only on one node but on all relevant nodes, right?
ref: 
https://github.com/openshift/cluster-node-tuning-operator/pull/746/files/23f53670b348a210197ec8d933c7db6fa48934b5#diff-ce7d5cbe8bfc6cc4efbf81602d5f238b37bafcbb68bf2c1ad66f1e1b148a33c5R163

Reply:
Addressed in the latest commit